### DISCUSSION TITLE
玩个小游戏 - 晦涩弗里曼的博客

### DISCUSSION THREAD

**Discussion by giscus** [obscurefreeman/blog#8] [Giscus]
# 玩个小游戏 - 晦涩弗里曼的博客

想要玩小游戏吗？

https://obscurefreeman.github.io/blog/%E6%8A%80%E6%9C%AF/%E7%8E%A9%E4%B8%AA%E5%B0%8F%E6%B8%B8%E6%88%8F/

**Comment by obscurefreeman** [comment]
**很有趣的游戏，我打到了九百万分，你能超过我吗？**
顺带一提，这个游戏一直往上走可能会卡住。
